migrations/20230830024631_cw721_activity_add_current_owner.ts

```diff
@@ -0,0 +1,48 @@
+import { Knex } from 'knex';
+import CW721Activity from '../src/models/cw721_tx';
+import _ from 'lodash';
+import { CW721_ACTION } from '../src/services/cw721/cw721.service';
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('cw721_activity', (table) => {
+    table.string('owner');
+  });
+  await knex.transaction(async (trx) => {
+    const activities = await CW721Activity.query()
+      .joinRelated('event')
+      .orderBy('event.block_height', 'asc')
+      .transacting(trx);
+    const done: CW721Activity[] = [];
+    activities.forEach((activity) => {
+      const latestActivity = _.findLast(done, function (e) {
```

Comment:
lưu 1 object latest theo contract_id vào token_id, khi for qua từng event thì update vào bảng đó, lúc cần lấy ra là được